pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
```

Comment:
The exercise brief didn't explain this, but you could try moving to poetry entirely and stop using pip/setup.sh.

If you install poetry on your machine, you should be able to run `poetry install` and `poetry run flask run`
Then you don't need to have both requirements.txt and pyproject.toml

If you make that change, don't forget to update the README.

Also, you will need to add test dependencies to this file, like pytest. They should go under the section `[tool.poetry.dev-dependencies]`